examples/protocol_specification_ex/fipa.yaml

```diff
@@ -0,0 +1,25 @@
+name: fipa
+author: fetchai
+version: 0.1.0
+license: Apache 2.0
+description: 'A protocol for FIPA ACL.'
+speech_acts:
+  cfp:
+    query: Query
+  propose:
+    proposal: list
+    price: float
+  accept_w_inform:
+    info: dict
+  match_accept_w_inform:
+    info: dict
+  inform:
+    info: dict
+  accept: {}
+  decline: {}
+  match_accept: {}
+
+  # Differences:
+  # propose -> proposal is a list of Descriptions. Declare each d in proposal is of type description.
```

Comment:
Why not include a Proposal special data type?

Reply:
Done

Reply:
Done